contentful.json

```diff
@@ -57,23 +57,23 @@
     "Client Side Forms": "No",
     "Plugin System": "No",
     "Customizable UI": "Yes",
-    "Custom Field Types": "No",
+    "Custom Field Types": "Yes",
     "User Management": "Yes",
     "Role Based Permissions": "Yes",
     "Document Level Permissions": "No",
-    "OAuth 2.0 Support": "No",
-    "Auditing": "No",
+    "OAuth 2.0 Support": "Yes",
+    "Auditing": "Yes",
     "API Keys": "Yes",
     "Project Support": "Yes",
     "I18N Localized Content": "Yes",
     "Content Trees": "No",
     "Tagging": "Yes",
-    "Content Relations": "Yes",
-    "Nesting of Fields": "No",
-    "GDPR Compliance": "No",
-    "Editing Conflict Detection": "No",
+    "Content Relations": "1:1, 1:n, n:n, n:1",
+    "Nesting of Fields": "Yes",
+    "GDPR Compliance": "Yes",
+    "Editing Conflict Detection": "Yes",
     "Versioning": "Yes",
-    "Content Scheduling": "No",
+    "Content Scheduling": "Yes",
```

Comment:
https://www.contentful.com/developers/docs/tutorials/general/scheduling-posts/
https://www.contentful.com/blog/2016/04/12/scheduled-publishing/

Reply:
I don't think it would be fair to add this to the list. Any other CMS which does not have core support for scheduling could use an external tool to trigger publishing of contents. For the other CMS I only listed this if the API supported this directly.

Reply:
Fair enough. I'll revisit this when Contentful has core support for content scheduling.